pdns/dnsdistdist/docs/changelog.rst

```diff
@@ -1,6 +1,85 @@
 Changelog
 =========
 
+.. changelog::
+  :version: 1.4.0-rc3
+  :released: 30th of September 2019
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 8083
+    :tickets: 7845
+
+    Clean up our interactions with errno
+
+  .. change::
+    :tags: Improvements, DNS over HTTPS, DNS over TLS
```

Comment:
You might want to add 'DNS over HTTPS' and 'DNS over TLS' to `changelog_inner_tag_sort` in `doc/conf.py`for sorting and categorization

Reply:
Done, thanks!